aruco_detect/src/aruco_detect.cpp

```diff
@@ -329,6 +329,10 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
         ROS_INFO("Detected %d markers", (int)ids.size());
 
         for (size_t i=0; i<ids.size(); i++) {
+	    if (std::count(ignoreIds.begin(), ignoreIds.end(), ids[i]) != 0) {
```

Comment:
I believe `std::find` is the canonical way to check for existence.

Consider using: `std::find(ignoreIds.begin(), ignoreIds.end(), ids[i]) != ignoreIds.end()`

Reply:
That code is directly copied from later on. Do you want me to change both?